RUFAS/simulation_engine.py

```diff
@@ -102,6 +106,9 @@ def _run_simulation_main_loop(self) -> None:
 
     def _daily_simulation(self) -> None:
         """Executes the daily simulation routines."""
+        process_degradations_today = self.time.current_julian_day % 15 == 0
+        if self.run_end_to_end_testing and process_degradations_today:
+            self.feed_manager.process_degradations(self.weather, self.time)
```

Comment:
```suggestion        
        if self.run_end_to_end_testing:
            process_degradations_today = self.time.current_julian_day % 15 == 0
            if process_degradations_today:
                self.feed_manager.process_degradations(self.weather, self.time)
```
gives a minor performance boost when e2e is not running.